### ISSUE TITLE
Questionable perf_hooks import

### ISSUE THREAD

**Issue by NeKzor** [source-lib/sourcelib#2]
Any reason why this is part of the library?
https://github.com/source-lib/sourcelib/blob/b58d4b01d9c97c562446e228580cac727d8d8720/packages/kv/src/tokenizer.ts#L7

Found this while testing the package in a different runtime which does not support this API yet but the usage of `timerify` seems very application specific.

**Comment by StefanH-AT** [comment]
You're right. I just ran into this myself. Shouldn't be part of the library output
